app/helpers/clubs_helper.rb

```diff
@@ -1,4 +1,9 @@
 module ClubsHelper
+  def club_previous_attendee_count(club)
+    previous_meeting = club.meetings.where(["created_at < ?", Date.today]).last
+    previous_meeting.blank? ? 'N/A' : previous_meeting.feedback_responses.length
+  end
```

Comment:
Could this be replaced by `@current_club.meetings.last.attendee_count` inline in the view?